lib/base-controller.js

```diff
@@ -59,11 +59,13 @@ BaseController.prototype.getErrorStep = function getErrorStep(err, req) {
 
 BaseController.prototype.locals = function controllerLocals(req, res) {
   var locals = Controller.prototype.locals.apply(this, arguments);
+  var meta = (this.options || {}).meta;
+
   return _.extend({}, locals, {
     baseUrl: req.baseUrl,
     nextPage: this.getNextStep(req, res),
     errorLength: getErrorLength.apply(this, arguments)
-  });
+  }, meta || {});
```

Comment:
sorry yeah that's what I was actually getting at. I just used a terrible collection of words.

Perhaps `this.options` should should be tested in the constructor so that we can make the line for meta a bit cleaner. it'd clean up line 28 also?